compiler/control/OMROptions.cpp

```diff
@@ -2761,8 +2762,8 @@ OMR::Options::jitPreProcess()
          }
       else if (TR::Compiler->target.is32Bit())
          {
-         // On 31-Bit Linux on Z, zNext exploitation is disabled by default.
-         self()->setOption(TR_DisableZNext);
+         // On 31-Bit Linux on Z, z17 exploitation is disabled by default.
+         self()->setOption(TR_DisableZ17);
```

Comment:
Actually this is z16 - I do not think 31-bit JVM was tested on Z16. Can I request you to change this to Z16 - once we confirm, we can update this.

Reply:
Done: https://github.com/eclipse-omr/omr/pull/7738/commits/3fe6894c5251d1651193fd6b3a5253d8781f54f4